source/material_model/rheology/strain_dependent.cc

```diff
@@ -569,6 +584,15 @@ namespace aspect
                               * this->get_timestep();
               break;
             }
+            case fracture_healing:
+            {
+              // Formula: APS_new = APS_current + delta_APS, where
+              // delta_APS = edot_ii*dt - recovery_rate*dt is the strain
+              // increment due to fracture strain healing. The recovery_rate
```

Comment:
1. What does APS stand for?
2. `delta_APS` is not just 'the strain increment due to fracture strain healing' according to the definition here. It also includes the increase in `strain edot_ii`*dt.
-> Please update the comment.